### ISSUE TITLE
No recording with the new version

### ISSUE THREAD

**Issue by LtJaeger** [Jabbah/OpenXR-Layer-OBSMirror#7]
I replaced 0.2.3, which works well, with 0.2.4 and no matter what I do, I don't get a "picture" with the OpenXRCapture. If I revert to the previous version, everything is fine again. I installed the files per instructions and allowed every file in the package

**Comment by Jabbah** [comment]
Can you make sure the plugin files in the OBS dir have been updated
correctly? The plugin changed as well this time so won't work with the old
files.

**Comment by LtJaeger** [comment]
Yes, I checked couple of times. Made sure I allowed them beforehand and copied them over, first the folder and then the files again. I believe I made no install error, but up for correction on this. Layer shows up correctly. Might it conflict with different layers? Got like a couple, like Fred's Openkneeboard and handtracking, Norberts XRNS and VRK.
